docs/git/index.md

```diff
@@ -4,48 +4,87 @@ title: Introduction
-4. Installed VS Code.
+_If you are a teacher reading this, please check the [Instructor Notes](./instructors)_
 
 ## What problem does Git solve?
 
-Git is used to solve many problems, but today we're going to focus on how it:
-* Helps us to share information.
-* Enables people to make their own changes to that information, and share it back.
-* Allows us to track how information has changed over time.
+Git is a version control tool, used to keep the history of changes, and make collaboration easier. It's used to solve many problems, but today we're going to focus on how it:
+
+- Helps us to share information.
+- Enables people to make their own changes to that information, and share it back.
+- Allows us to make checkpoints so that we can save our work as we go along.
+- Allows us to track how information has changed between each checkpoint, and go back to older versions of our work if we want to see what we tried before, or to undo changes.
+- Enables us to try more things, because if they didn't work out, we can always go back to what we had working before, by going back to a checkpoint.
+
+### Versioning
+
+We use Git to track different versions of files.
+
+You've probably in the past saved a document in a file with "draft" in its name, and then another with "version 1" in its name, and eventually one called "final", and then "really final", and then "final after feedback". It can be hard to know what the latest one is, and to track what order the files came in. But we keep these files around, because they were useful, and we may want to check something from them.
+
+Git helps us to avoid this problem. Before we see how Git helps, let's try an exercise:
+
+#### Exercise 1 (10 minutes)
+
+Open these three links - they are different stages of a draft blog post about CodeYourFuture:
+
+TODO: Move to CYF repo
 
-## How does Git do that?
+- [final](https://illicitonion.github.io/git-draft-blog-post-example/final)
+- [revised](https://illicitonion.github.io/git-draft-blog-post-example/revised)
+- [v1](https://illicitonion.github.io/git-draft-blog-post-example/v1)
+
+Try to find all of the differences between these three documents. Can you work out which one is the finished one which was meant to go on the website?
+
+You probably found it hard to see all of the differences (some were really small, like adding or removing a comma!), and non-obvious which is the most complete version!
+
+#### Exercise 2 (10 minutes)
+
+Get into groups of three. One of your group should make a shopping list of 5 items, and send the list to the group.
+
+The other two people in the group should each add a few items to the list, and also remove one item, and then send back the whole new list to the person who made the list.
+
+The first person should merge the new lists, so that there is one shopping list. The items people removed should not be in the final list.
```

Comment:
Maybe the group should report back to the class what their experience was like? Ensures every has the same takeaway from the exercise.